site/ja/docs/privacy-sandbox/user-agent/snippets/index.md

```diff
@@ -0,0 +1,170 @@
+    Lin: 'Linux; Android 10; K',
+    Win: 'Windows NT 10.0; Win64; x64',
+    Mac: 'Macintosh; Intel Mac OS X 10_15_7',
+    'X11; C': 'X11; CrOS x86_64',
+    'X11; L': 'X11; Linux x86_64',
+  };
+
+  const reducedUA =
+    `Mozilla/5.0 (${unifiedPlatform[matched.groups.platform]}) ` +
+    `AppleWebKit/537.36 (KHTML, like Gecko) ` +
+    `Chrome/${matched.groups.major}.0.0.0${matched.groups.mobile} Safari/537.36`;
+  // navigator.userAgent を情報量削減後の文字列でオーバーライドする
+  Object.defineProperty(navigator, 'userAgent', {
+    value: reducedUA,
+    writable: false,
+    configurable: true,
+  });
+}
+```
+
+### Node.js
+
+[Node.js](https://nodejs.org/) で受信した [User-Agent ヘッダー](https://developer.mozilla.org/docs/Web/HTTP/Headers/User-Agent)を情報量削減後の形式でオーバーライドするには、次のスニペットを使用します。
+
+デモ: [reduced-ua.glitch.me/server-side](https://reduced-ua.glitch.me/server-side)
+
+```js
+const chromeUAs = /^Mozilla\/5\.0 \(((?<platform>Lin|Win|Mac|X11; C|X11; L)+[^\)]+)\) AppleWebKit\/537.36 \(KHTML, like Gecko\)Chrome\/(?<major>\d+)[\d\.]+(?<mobile>[ Mobile]*) Safari\/537\.36$/;
+const matched = chromeUAs.exec(request.get(\'user-agent\'));
+
+if (matched) {
+  const unifiedPlatform = {
+    'Lin': 'Linux; Android 10; K',
+    'Win': 'Windows NT 10.0; Win64; x64',
+    'Mac': 'Macintosh; Intel Mac OS X 10_15_7',
+    'X11; C': 'X11; CrOS x86_64',
+    'X11; L': 'X11; Linux x86_64',
+  };
+  request.headers['user-agent'] = `Mozilla/5.0 (${unifiedPlatform[matched.groups.platform]}) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/${matched.groups.major}.0.0.0${matched.groups.mobile} Safari/537.36`;
+}
+```
+
+### Cloudflare Workers
+
+[Cloudflare Workers](https://developers.cloudflare.com/workers/) を使用すると、サイトへのリクエストをプロキシして変換できます。次のスニペットは、受信した [User-Agent ヘッダー](https://developer.mozilla.org/docs/Web/HTTP/Headers/User-Agent)を情報量削減後の形式に置き換えます。
+
+```js
+addEventListener('fetch', event => {
+  event.respondWith(handleRequest(event.request));
+});
+
+async function handleRequest(request) {
+  const chromeUAs =
+    /^Mozilla\/5\.0 \(((?<platform>Lin|Win|Mac|X11; C|X11; L)+[^\)]+)\) AppleWebKit\/537.36 \(KHTML, like Gecko\) Chrome\/(?<major>\d+)[\d\.]+(?<mobile>[ Mobile]*) Safari\/537\.36$/;
+  const matched = chromeUAs.exec(request.headers.get('user-agent'));
+
+  if (matched) {
+    const unifiedPlatform = {
+      Lin: 'Linux; Android 10; K',
+      Win: 'Windows NT 10.0; Win64; x64',
+      Mac: 'Macintosh; Intel Mac OS X 10_15_7',
+      'X11; C': 'X11; CrOS x86_64',
+      'X11; L': 'X11; Linux x86_64',
+    };
+
+    const clonedRequest = new Request(request, {referrer: request.referrer});
+    clonedRequest.headers.set(
+      'user-agent',
+      `Mozilla/5.0 (${
+        unifiedPlatform[matched.groups.platform]
+      }) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/${
+        matched.groups.major
+      }.0.0.0${matched.groups.mobile} Safari/537.36`
+    );
+    return await fetch(clonedRequest);
+  } else {
+    return await fetch(request);
+  }
+}
```

Comment:
Please replicate the original English version as much as possible (except the Japanese translation).